modules/gmake2/gmake2_cpp.lua

```diff
@@ -425,6 +432,13 @@
 		p.outln('ALL_CXXFLAGS += $(CXXFLAGS) $(ALL_CPPFLAGS)' .. flags)
 	end
 
+	function cpp.clangtidy(cfg, toolset)
+		if cfg.clangtidy ~= nil then
```

Comment:
"Off" (or false if there is some translation phases for that parameter) would wrongly enter in that case.